pkgs/by-name/et/eta/package.nix

```
@@ -0,0 +1,33 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eta";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "aioobe";
+    repo = "eta";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-UQ8ZoxFAy5dKtXTLwPolPMd7YJeEjsK639RkGCMY6rU=";
+  };
+
+  outputs = [
+    "out"
+    "man"
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

```suggestion
  makeFlags = [ "PREFIX=$(out)" "CC=cc" ];
```

Otherwise it's gcc and will fail on darwin